docs/publishers/events.rst

```diff
@@ -0,0 +1,48 @@
+Custom DOM events
+=================
+
+Hypothesis client emits a few custom DOM events to which your app can react.
```

Comment:
Related to this, there is an existing issue about documenting the `scrolltorange` event. Before we do that though, we should change it to have a `hypothesis:` prefix. We may also need to keep the old event for backwards compatibility for a period of time.